src/matrix/storage/idb/Store.ts

```diff
@@ -131,7 +131,7 @@ export class Store<T> extends QueryTarget<T> {
     private _transaction: Transaction;
 
     constructor(idbStore: IDBObjectStore, transaction: Transaction) {
-        super(new QueryTargetWrapper<T>(idbStore));
```

Comment:
can we bump the `_transaction` property to `QueryTarget` and keep the `IDBKeyRange` getter and add one for the factory? That way we don't create too many new fields...